pandas/core/frame.py

```diff
@@ -4486,6 +4489,12 @@ def _series_round(s, decimals):
                 return s.round(decimals)
             return s
 
+        validate_args(args, min_length=0, max_length=1)
+        out = args[0] if args else None
+
+        if out is not None:  # Series.round has justification
```

Comment:
Added a `_validate_out_arg` to `validators.py`.  From an organization perspective, it seems logical to put all of these argument validators in one place IMO.